src/app/file-manager/services/limit-transferables.service.ts

```diff
@@ -50,14 +51,20 @@ export class LimitTransferablesService {
     }
 
     maxFiles.forEach(item => {
-      item.status = status;
-      this.store.dispatch(new Transferables.UpdateItem(item));
+      if (item.type !== Type.EXPORT_S3) {
+        item.status = status;
+        this.store.dispatch(new Transferables.UpdateItem(item));
+      }
     });
 
     if (type === Type.DOWNLOAD) {
       this.gcsService.downloadFiles(maxFiles);
     } else if (type === Type.UPLOAD) {
       this.gcsService.uploadFiles(localStorage.getItem('uploadBucket'), maxFiles);
+    } else if (type === Type.EXPORT_S3) {
+      maxFiles[0].status = ItemStatus.EXPORTING_S3;
```

Comment:
Not sure why this code is using `maxFiles[0]`, do we know that there can only be one? A comment would be helpful here.